element-messenger/src/ElementFrame.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="element-embed">
+    <div v-if="configuration.element && configuration.element.server" id="element-frame-container">
+      <iframe id="element-frame" :src="configuration.element.server"></iframe>
+    </div>
+    <div v-if="!configuration.element || !configuration.element.server">
+      {{$gettext('Unfortunately Element was not configured correctly.')}}
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'ElementFrame',
+  data: () => ({
+    configuration: window.phoenixConfig || []
```

Comment:
Access pls config via the store. We want to get rid of the `window.phoenixConfig` object.